CMakeLists.txt

```diff
@@ -82,6 +82,22 @@ if(NOT DEFINED PLUGIN_LIBRARY_OUTPUT_DIRECTORY)
 endif()
 message(STATUS "${CMAKE_PROJECT_NAME}: PLUGIN_LIBRARY_OUTPUT_DIRECTORY: ${PLUGIN_LIBRARY_OUTPUT_DIRECTORY}")
 
+# When building, don't use the install RPATH already
+# (but later on when installing)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
```

Comment:
You explain what it does, but why not have rpath available in the build directory?

Reply:
True, we don't want our CMAKE_INSTALL_PREFIX to be in CMAKE_BUILD_RPATH. I guess, running from build directory might be unusable also because it doesn't have a structure that JANA_PLUGIN_PATH can point to. Thanks for explaining. Marking this resolved.